src/ClientData/include/ClientData/ApiTrackValue.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CLIENT_DATA_API_TRACK_VALUE_H_
+#define CLIENT_DATA_API_TRACK_VALUE_H_
+
+#include <cstdint>
+#include <string>
+#include <utility>
+
+namespace orbit_client_data {
+
+// Represents a value tracking event from our manual instrumentation API (Orbit.h). It contains a
+// name of the track to which the data point should be added, and the data point itself.
+// Note that we only display "double" values in the UI. Values of other types need to be converted
+// to double.
+// See `orbit_grpc_protos::ApiTrackValue*` protos.
+class ApiTrackValue {
+ private:
+ public:
+  ApiTrackValue(const uint32_t& process_id, const uint32_t& thread_id, const uint64_t& timestamp_ns,
```

Comment:
I'm not sure if you have been doing with the other protos, but it might be worth to explicitly delete the default constructor so that it's clearer that it shouldn't be added for convenience at a later point.

Reply:
Good point. We have not. But I did it at least for this one.